### DISCUSSION TITLE
Categories in Allure Report

### DISCUSSION THREAD

**Discussion by matheusbarcelos** [allure-framework/allure2#2070] [General Discussion]
Hello.

I want to generate the Allure Report separated by categories. Basically separate by Android and IOS.
I'm using Ruby and Cucumber.

Kindly, has anyone developed something similar? Is it possible to implement?

**Comment by PavelMal** [comment]
@matheusbarcelos 
Good day!
What's the purpose of it?
You can simple use different suits for it or use annotations @Story/@Epic
1. Example with stories
<img width="720" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/fcc54f39-fa74-4e4c-a456-44acbadc4bc8">


2. Example with Suite
<img width="730" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/f022a93a-7037-4ca9-95f0-5aac069d1eb9">
